src/api/github/index.ts

```diff
@@ -19,7 +17,7 @@ function defaultErrorHandler(error) {
 
 githubEvents.onError(defaultErrorHandler);
 
-function makeUserTokenClient(token: string = GH_USER_TOKEN) {
+function makeUserTokenClient(token: string) {
```

Comment:
Should this function be removed an instead require folks to use the src/spi/github/makeUserTokenClient file instead?

Reply:
Oops! Good catch, I was going back and forth. Gone in bf33f8094762254cac446c5eec1c6e54ea737800.